.github/workflows/publish_draft_releases.yml

```diff
@@ -75,9 +73,7 @@ jobs:
   build-node-current:
     timeout-minutes: 180
     if: ${{ startsWith(github.ref, 'refs/tags/v') }}
```

Comment:
lets test this action by tagging this branch

Reply:
+1 , i'd also test the already existing manual trigger jobs.

Reply:
yeah, i think we have to merge to `manta` branch first to test this, or the tags will just run the `manta` branch pipeline (which uses ondemand vm)

Reply:
ya failed to run when I created tag, maybe will work off manta branch idk

Reply:
actually you might be right let me try

Reply:
yeah it didn't work haha, this has to be merged to main to test